governance/MEMBERSHIP_SURVEY.md

```diff
@@ -0,0 +1,32 @@
+
+# Membership Survey
+
+Surveys for current and past members to collect data on member wants and needs. following #375 issue.
+
+## Suggested Questions
+
+@amiller-gh
+
+What is your current happiness (perceived value?) of Ind. Membership program?
+Net promoter score – how likely would you recommend the program to a friend?
```

Comment:
You can remove `Net promoter score` here since it's just describing what the question is commonly referred to:

```suggestion
How likely would you recommend the program to a friend?
```